packages/shared/package.json

```diff
@@ -2,6 +2,14 @@
   "name": "@saleor/apps-shared",
   "version": "1.12.2",
   "type": "module",
+  "exports": {
+    ".": {
+      "default": "./index.ts"
+    },
+    "./compose": {
+      "default": "./src/compose.ts"
+    }
```

Comment:
1. Im not sure if you need "default"? You can check the spec
2. Maybe we dont have to manually define every export? Is it possible to only target `".": "./src"` or something like this, so every file will be accessbile?

Reply:
I've added & migrated all apps to use new export with what you suggested 😄